plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -873,6 +873,52 @@ fu_synaptics_rmi_device_write_firmware (FuDevice *device,
 	/* erase all */
 	//FIXME: write $3 into F34_Flash_Data3
 	//FIXME: wait for ATTN and check success
+	/* v7+ */
```

Comment:
This comment makes me wonder on what happens in behavior for v6 and less?  Should those be setting an error about failing to flash?  Or something in coldplug about how they're not supported?

Reply:
The erasing code for BLv7- doesn't be implemented, and it would be added later.

Reply:
That's fine - can you please detect v6 and less and just set an error for now then?